receiver/dockerstatsreceiver/metadata.yaml

```diff
@@ -132,19 +132,20 @@ metrics:
       monotonic: true
       aggregation_temporality: cumulative
   container.cpu.utilization:
-    enabled: false
+    enabled: true
     description: "Percent of CPU used by the container."
     unit: "1"
-    warnings:
-      if_enabled_not_set: This metric will be enabled by default in v0.82.0.
     gauge:
       value_type: double
   container.cpu.percent:
-    enabled: true
-    description: "Deprecated: use `container.cpu.utilization` metric instead. Percent of CPU used by the container."
+    enabled: false
+    description: "[DEPRECATED] Use `container.cpu.utilization` metric instead. Percent of CPU used by the container."
     unit: "1"
     warnings:
-      if_enabled: This metric will be disabled in v0.82.0 and removed in v0.85.0.
+      if_configured: >-
+        The metric is deprecated and will be removed in v0.86.0. Please use `container.cpu.utilization` instead. See
```

Comment:
```suggestion
        The metric is deprecated and will be removed in v0.88.0. Please use `container.cpu.utilization` instead. See
```

Reply:
Sorry I forgot the metadata's version correction 🤦. I's properly updated here: [1f4c199](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/24183/commits/1f4c199b760232f29f37d350c14a9560ff41737f). Thanks!